doc/content/documentation/namespaces.md

```diff
@@ -11,12 +11,48 @@ Every Phel file is required to have a namespace. A valid namespace name starts w
 (ns name imports*)
 ```
 
-Defines the namespace for the current file add required imports to the environment. Imports can either be _uses_ or _requires_. The keyword `:use` is used to import PHP classes and the keyword `require` is used to import Phel modules.
+Defines the namespace for the current file and adds imports to the environment. Imports can either be _uses_ or _requires_. The keyword `:use` is used to import PHP classes and the keyword `require` is used to import Phel modules.
 
 ```phel
 (ns my\custom\module
-  (:use Some\Php\Class)
+  (:use \Some\Php\Class)
```

Comment:
Actually the `\` at the beginning is not required. Therefore I would omit it in the examples.